modules/rules_browsers/0.1.0/presubmit.yml

```diff
@@ -0,0 +1,59 @@
+matrix:
+  platform:
+    - debian11
+    - ubuntu2204
+    - macos
+    - macos_arm64
+  bazel:
+    - 8.x
+    - 7.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    # Building targets in the anonymous test module does not work because it
+    # does not allow customization of .bazelrc.
+    # The module uses `aspect_rules_ts`, so there are some options that need to
+    # be set.
+    #
+    #     common --@aspect_rules_ts//ts:skipLibCheck=always
```

Comment:
you can use build_flags (see example https://github.com/bazelbuild/bazel-central-registry/blob/21c2172b0f8437d6eebca80f8a633ccd95f5a206/modules/flip/1.4/presubmit.yml#L9)